tests/cc13b/input.dat

```diff
@@ -0,0 +1,56 @@
+#! Tests RHF CCSD(T)gradients
+memory 256 mb
+
+molecule h2o {
+0 1
+O
+H 1 R
+H 1 R 2 A
+
+R = 1.0
+A = 104.5
+}
+
+set {
+   reference rhf
+   basis 6-31g 
+   r_convergence 13
+   e_convergence 13
+   d_convergence 13
+}
+
+analytic, wfn = gradient('ccsd(t)', return_wfn=True)
+ref_vals = [
+             [ 0.00000000000000,     0.00000000000000,    -0.02788939642641],  # FINITE DIFFERENCE 5 POINT FORMULA
+             [ 0.00000000000000,    -0.00904503210340,     0.01394469821321],
+             [ 0.00000000000000,     0.00904503210340,     0.01394469821321]
+           ]
+fin_diff = psi4.Matrix(3, 3)
+fin_diff.set(ref_vals)
+compare_matrices(analytic, fin_diff, 10, "1. RHF-CCSD(T) gradient comparison with finite differences") #TEST
```

Comment:
I agree, it _looks_ like it should work. But all the same, I don't think it will over lots of architectures. Go for 8 if you like, and we'll see how it fares in the wild.